alphalens/performance.py

```diff
@@ -60,21 +57,17 @@ def src_ic(group):
     factor_data = factor_data.copy()
 
     grouper = [factor_data.index.get_level_values('date')]
+
     if by_group:
         grouper.append('group')
 
-    if group_adjust:
-        factor_data = utils.demean_forward_returns(factor_data,
```

Comment:
I'm confused, these aren't the ICs, right? So de-meaning should make a difference here.

Reply:
Oh I get it, it doesn't matter for computing the IC which is what we're doing here.

Reply:
yes, if apply  utils.demean_forward_returns to the returns the resulting IC is the same as not applying utils.demean_forward_returns